projects/morpho-blue/index.js

```diff
@@ -28,6 +29,7 @@ Object.keys(config).forEach((chain) => {
           abi: abi.morphoBlueFunctions.idToMarketParams,
         })
       )
+        .filter((i) => !blackList.includes(i.collateralToken.toLowerCase()))
         .map((i) => [i.collateralToken, i.loanToken])
         .flat();
       return api.sumTokens({ owner: morphoBlue, tokens });
```

Comment:
sumTokens method support `blacklistedTokens`  param that takes care of token case, like this: https://github.com/DefiLlama/DefiLlama-Adapters/blob/main/projects/smartcredit.js#L91